CONTRIBUTING.md

```diff
@@ -2,27 +2,25 @@
 
 We ♥ contributors! By participating in this project, you agree to abide by the Ruby for Good [code of conduct].
 
-**First:** if you're unsure or afraid of *anything*, just ask or submit the issue or pull request anyways. You won't be yelled at for giving your best effort. The worst that can happen is that you'll be politely asked to change something. We appreciate any sort of contributions, and don't want a wall of rules to get in the way of that.
+If you're unsure about an issue or have any questions or concerns, just ask in an *existing issue* or *open a new issue*. If you don't have any questions and the issue is clear and not currently assigned to anyone, you can open a PR as well. You won't be yelled at for giving your best effort. The worst that can happen is that you'll be politely asked to change something. We appreciate any sort of contributions, and don't want a wall of rules to get in the way of that.
 
-[code of conduct]: https://github.com/rubyforgood/code-of-conduct
-
-Here are the basic steps to submit a pull request. Make sure that you're working on an [open issue]–if the relevant issue doesn't exist, open it!
+Here are the basic steps to submit a pull request.
 
-[open issue]: https://github.com/rubyforgood/casa/issues
+1. Claim an issue on [our issue tracker][issues] by assigning it to yourself (core team member) or commenting. If the issue doesn't exist yet, open it.
 
-1. Claim an issue on [our issue tracker][open issue] by assigning it to yourself (core team member) or commenting. If the issue doesn't exist yet, open it.
+2. Fork the [repo] and clone your forked repo locally on your machine.
```

Comment:
If you change all the 2. 3. 4. to `1.` then it will auto-increment for you which is more convenient

Reply:
Lol an ex co-worker used to do that and I always wondered why until I was re-numbering them just now but was so committed to my old ways that did not thing to change. Updated.